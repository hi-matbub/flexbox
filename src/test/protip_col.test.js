import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ProTipCol from '../comps/protip_col.jsx';

it('renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<ProTipCol />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders snapshot as expected', () => {
  const tree = renderer.create(<ProTipCol />).toJSON();
  expect(tree).toMatchSnapshot();
});
