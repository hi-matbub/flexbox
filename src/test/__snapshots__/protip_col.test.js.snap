// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot as expected 1`] = `
<div
  className="pro-tips-col"
>
  <span
    className="pro-tips-title"
  >
    <h2>
      Try the Flexbox Generator!
    </h2>
  </span>
</div>
`;
